cola/widgets/remote.py

```diff
@@ -680,9 +689,45 @@ def push_to_all(self, _remote, *args, **kwargs):
         """Push to all selected remotes"""
         selected_remotes = self.selected_remotes
         all_results = None
+
+        pushed_remotes = 0
+
         for remote in selected_remotes:
             result = self.model.push(remote, *args, **kwargs)
+
+            if result[0] == 0:
+                pushed_remotes = pushed_remotes + 1
+
             all_results = combine(result, all_results)
+
+        if prefs.notify_on_push(self.context):
+            try:
+                notification = Notify()
+
+                total_remotes = len(selected_remotes)
+
+                if pushed_remotes == total_remotes:
+                    notification.title = N_('Push Successful')
+                    notification.icon = resources.package_data('icons', 'git-cola.svg')
+                elif pushed_remotes != 0:
+                    notification.title = N_('Push Partially Successful')
+                    notification.icon = resources.package_data(
+                        'icons', 'git-cola-error.svg'
+                    )
+                else:
+                    notification.title = N_('Push Failed')
+                    notification.icon = resources.package_data(
+                        'icons', 'git-cola-error.svg'
+                    )
+
+                notification.message = N_(
+                    f'{pushed_remotes}/{total_remotes} remotes updated'
+                )
```

Comment:
This should be the title not the message. The message should mention the remote names that are looped over in the `remote` variable as being either failed being "Not pushed" or "Pushed".

If there are no "Not pushed" remotes then we shouldn't see that part of the message and it should only mentioned the "Pushed" repos. Likewise, if there are no "Pushed" remotes then we shouldn't see that part of the message and only the "Not pushed" section should be present.